spec/workspace-spec.js

```diff
@@ -116,6 +118,63 @@ describe('Workspace', () => {
         expect(atom.workspace.getTextEditors().length).toBe(0)
       })
     })
+
+    describe('where a dock contains an editor', () => {
+      afterEach(() => {
+        atom.workspace.getRightDock().paneContainer.destroy()
+      })
+
+      it('constructs the view with the same panes', () => {
```

Comment:
Can we do that separately? This file already has > 100 usages of `waitsForPromise()` and I feel like it's better to maintain the consistency and switch them all at once.